### PR TITLE
Update podspec platform support to match SwiftPM.

### PR DESCRIPTION
**Pull request by Zandor300**
The SwiftPM `Package.swift` has specified tvOS support but this isn't also specified for Cocoapods. This PR will match up the 2 so both specify tvOS support.

@BastiaanJansen Could you take a look?

## Discussion

_~~What about visionOS support? Haven't tested using this library on visionOS yet (would like to) but maybe we could add visionOS as a supported platform to both SwiftPM and Cocoapods if it works out of the box?~~_

_~~Or maybe that should be out of scope for this PR and should be handled in a new PR, but wanted to open the discussion.~~_

Edit: visionOS support was added, but support for iOS 11 was dropped in this PR.

### CODE REVIEW

**Review by BastiaanJansen — DISMISSED** [review]

**Comment by BastiaanJansen** [comment]
I agree we should add VisionOS as a supported platform. It's fine if you include this in this PR.

**Comment by Zandor300** [comment]
To specify visionOS support in Package.swift, we need to upgrade the `swift-tools-version` from `5.3` to `5.9`.

But doing that would result in the following warning for iOS:

```
Warning: 'v11' is deprecated: iOS 12.0 is the oldest supported version
```

I personally think deprecating support for iOS 11 is fine, but you may think differently.

I don't think a lot of current apps still support iOS 11 and all devices that can run iOS 11 can upgrade to iOS 12 as can be seen here: https://iosref.com/ios (iOS 11 is never mentioned as "Max iOS", meaning iOS 11 devices can upgrade to iOS 12)

*Tl;dr* I've added visionOS support in one commit and increased the min iOS version in another. You could merge this PR without my iOS version bump commit if you choose to keep it on iOS 11.

**Comment by BastiaanJansen** [comment]
I think it is OK to drop support for iOS 11.
